rsub.py

```diff
@@ -186,7 +189,9 @@ def on_close(self, view):
 
 
 def plugin_loaded():
-    global SESSIONS, server
+    global SESSIONS, WINDOW_HANDLE, server
+
+    WINDOW_HANDLE = 'sublime_text.Sublime_text'
```

Comment:
Hm, plugin_loaded() is invoked for both Sublime 2 & 3. This will always overwrite WINDOW_HANDLE with "sublime_text.Sublime_text"?